src/main/kotlin/no/nav/familie/ef/sak/sigrun/ekstern/SigrunClient.kt

```diff
@@ -15,6 +16,23 @@ class SigrunClient(
     @Qualifier("azure") restOperations: RestOperations,
 ) : AbstractRestClient(restOperations, "sigrun") {
 
+    fun hentPensjonsgivendeInntekt(fødselsnummer: String, inntektsår: Int): PensjonsgivendeInntektResponse {
+        val uri = UriComponentsBuilder.fromUri(uri).pathSegment("api/sigrun/pensjonsgivendeinntekt")
+            .queryParam("inntektsaar", inntektsår.toString())
+            .build().toUri()
+
+        try {
+            val response = postForEntity<PensjonsgivendeInntektResponse>(uri, PersonIdent(fødselsnummer))
+            secureLogger.info("Pensjonsgivende inntekt for inntektsår $inntektsår: $response") // Fjernes når det er litt mer kjennskap til dataene
+            return response
+        } catch (e: HttpClientErrorException.NotFound) { // Kan få 404 selv om personen finnes, men dersom vedkommende ikke har registrert inntekt for gitt inntektsår
```

Comment:
Her er årsaken til at jeg misliker bruken av "notFound" når personen ikke har inntekt registrert: 
Det blir veldig vanskelig for oss å vite om det er 
- "Not found" - noen har endret apiet og urlen vi går mot finnes ikke lenger 
- "Not found" - personen har ikke inntekt


Det første ønsker vi jo å agere på, det siste er "ok" og kunne returnert et objekt tilsvarende "tom liste" eller noe :)

Reply:
Er så enig. Endte med å håndtere 404 i ef-proxy i stedet, så jeg fjernet denne catchen her. Er selvfølgelig ulempe at det ikke skilles på om personen ikke har inntekt eller at url er endret. Ser at det sendes med en feilmeldingskode i responsen dersom det mangler inntekt, så det er mulig det går an å håndtere dette og sjekke på meldingen i ef-proxy for å avgjøre om det skal returneres en tom liste eller kaste feilen videre

Reply:
Mulig det blir plukket opp av saksbehandlere hvis alle har 0 i inntekt dersom det går mot feil endepunkt